docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -12,7 +12,7 @@
   * [Spring Webflux](https://docs.spring.io/spring/docs/current/spring-framework-reference/web-reactive.html) 5.x (Optional²)
   * [Undertow](http://undertow.io/)  1.3.0.Final -> 2.0.27.Final
   * [RESTEasy](https://resteasy.github.io/)  3.1.0.Final -> 3.7.0.Final
-  * [Play Framework](https://www.playframework.com/) 2.6.x -> 2.7.x (Optional²)
+  * [Play Framework](https://www.playframework.com/) 2.6.x -> 2.8.x
```

Comment:
You should remove the play from here, https://github.com/apache/skywalking/blob/master/docs/en/setup/service-agent/java-agent/README.md#optional-plugins